app/Filament/Admin/Resources/ProgramResource.php

```diff
@@ -25,13 +25,16 @@ public static function form(Form $form): Form
     {
         return $form
             ->schema([
-                Forms\Components\TextInput::make('name')
-                    ->required()
-                    ->maxLength(255),
-                Forms\Components\TextInput::make('description')
-                    ->maxLength(255),
-                Forms\Components\TextInput::make('note')
-                    ->maxLength(255),
+                Forms\Components\Section::make('Program Details')
+                    ->schema([
+                        Forms\Components\TextInput::make('name')
+                            ->required()
+                            ->maxLength(255),
+                        Forms\Components\TextInput::make('description')
```

Comment:
255 characters is a little small for a description; I would typically go for a textarea entry and text field type for things like "description", "notes" etc.

Reply:
As discussed, I will update this later.